src/test/resources/com/puppycrawl/tools/checkstyle/sariflogger/ExpectedSarifLoggerDoubleError.sarif

```diff
@@ -0,0 +1,66 @@
+{
+  "$schema": "https://raw.githubusercontent.com/oasis-tcs/sarif-spec/master/Schemata/sarif-schema-2.1.0.json",
```

Comment:
please update our pom.xml to use https://github.com/groupon/json-schema-validator
and validate such json files by this schema, prevent any regression from further updates in such files, as this generation is hand crafted, there might be problems that we can not notice in review by human.
Let tool to guard us.

Reply:
Good call. Added the plugin.